### ISSUE TITLE
Execution failed for task ':expo:compileDebugJavaWithJavac' on Expo SDK 50

### ISSUE THREAD

**Issue by quantmvo-kms** [jamsch/expo-speech-recognition#7]
Hi @jamsch! My another project using Expo version 50.0.17. I'm trying to integrate with this library but got the following error when running on Android (it worked well on iOS):
```shell
> Task :expo:compileDebugJavaWithJavac FAILED
/Users/quantmvo/<my-project-path>/node_modules/expo/android/build/generated/expo/src/main/java/expo/modules/ExpoModulesPackageList.java:27: error: cannot find symbol
            expo.modules.speechrecognition.ExpoSpeechRecognitionModule.class,
                                          ^
  symbol:   class ExpoSpeechRecognitionModule
  location: package expo.modules.speechrecognition
1 error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':expo:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.
```
After searching solutions on the Internet, I came to [this issue](https://github.com/expo/expo/issues/28784) with a similar error as I got. I'm wondering if your project supports Expo SDK 50. If not, is there a way so that I can use your library perfectly without changing the SDK version?

**Comment by jamsch** [comment]
Hi @quantmvo-kms I've just released a version for SDK 50 under 0.1.12. You can type `npm install @jamsch/expo-speech-recognition@sdk-50` to install it. Let me know if it works okay.

**Comment by quantmvo-kms** [comment]
I've used that version and it worked on both iOS and Android. However, I found some other problems. I will compare the two versions `0.2` and `0.1` to verify the issues I'm facing and report new issues to you. Thanks again for your great support!
